test/day20Sorting.js

```diff
@@ -0,0 +1,36 @@
+const getOrder = (x, y, array) => {
+  let orderArray = array
+  if (array[y] > array[x]) {
+    const Menor = orderArray[x]
+    orderArray[x] = orderArray[y]
+    orderArray[y] = Menor
+    return orderArray
+  }
+}
+
+const getResult = (length, array) => {
+  let moves = 0
+  let orderArray = []
+  for (let i = 0; i < length; i++) {
+    for (let j = 0; j < length - 1; j++) {
+      const order = getOrder(j + 1, j, array)
```

Comment:
xq no usar built in sort function? 
https://www.w3schools.com/jsref/jsref_sort.asp

tambien puede pasar su funcion...